models/yang/annotations/openconfig-acl-annot.yang

```diff
@@ -33,6 +33,7 @@ module openconfig-acl-annot {
       deviate add {
```

Comment:
180 files? Why are NAT, Interfaces and ACL changes are shown in this pull request? Please open a new  pull request with only changes related to this feature.